platform/lifecycle_inputs.go

```diff
@@ -235,6 +237,14 @@ func envOrDefault(key string, defaultVal string) string {
 	return defaultVal
 }
 
+func sliceEnv(k string) str.Slice {
+	envVal := os.Getenv(k)
+	if envVal != "" {
+		return strings.Split(strings.ReplaceAll(envVal, " ", ""), ",")
```

Comment:
What's the idea behind replacing the spaces? Is this in one of the specs?

Reply:
This is been done if I remember correctly to avoid things like `--flatten=something, something, something`.
So in this way, we can either provide this format or this one `--flatten=something,something,something`.
Is this something we need to document in the spec? 🤔 At the end of the day we are not changing the current behavior, we are just preventing other formats from being passed through and not working properly.

Reply:
I don't have a problem with us helping out end users - but I think it being stuffed into `sliceEnv` is unexpected. It would break a rather innocent looking example like `--labels=red,granny smith,green` down the road. If we want to ensure the specific env doesn't have spaces, I think we should do that outside of this helper. Does that make sense?